### PR TITLE
Jagadeesh_Assignment_17_85min

### PR DESCRIPTION
**Pull request by JagadeeshMJun20**
image

image

### CODE REVIEW

File: src/jagadeeshMJun20/FirstAndLastIndex.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+		void printCharFreq(String name) {
+			for(int index=0; index<name.length(); index++) {
+				char ch = name.charAt(index);//t
+				int count = 0;
+				if (index == name.indexOf(ch)) {
+				     for(int i=0; i<name.length(); i++) {
+					 if (ch == name.charAt(i)) 
+						count++;
+						 if (count>1) {
+								int num = name.indexOf(ch);
```
Comment (KrishnaKTechnocredits) [inline src/jagadeeshMJun20/FirstAndLastIndex.java]: You have increased the complexity in code, Using one for loop and one if block you can achieve the output. Think once !!

File: src/jagadeeshMJun20/PreDefinedString.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+public class PreDefinedString {
+	void displayAllChars(String input) {
+		int digitCount=0,lowerCount=0,upperCount=0,specialCount=0,letterCount=0;
+		for(int index=0;index<input.length();index++) {
+			char ch = input.charAt(index);
+		if(	Character.isDigit(ch)) 
+			digitCount++;
+		else if(Character.isAlphabetic(ch)) {
+			letterCount++;
+			if(Character.isLowerCase(ch))
```
Comment (KrishnaKTechnocredits) [inline src/jagadeeshMJun20/PreDefinedString.java]: Without using letterCount variable and else if(Character.isAlphabetic(ch)) block you are able to achieve the expected output.

**Review by JagadeeshMJun20 — COMMENTED** [review]
Please review the changes as per comments provided.

**Review by JagadeeshMJun20 — COMMENTED** [review]
Added Code for Assignment dated 26th June. Please review

**Pull request merged**